locales/template/translation.json

```diff
@@ -428,15 +428,7 @@
   "Start typing a project": "",
   "Started": "",
   "Started <i>{{surveyName}}</i> survey": "",
-  "Started {{count}} day ago": "",
-  "Started {{count}} day ago_plural": "",
-  "Started {{count}} hour ago": "",
-  "Started {{count}} hour ago_plural": "",
-  "Started {{count}} minute ago": "",
-  "Started {{count}} minute ago_plural": "",
-  "Started {{count}} second ago": "",
-  "Started {{count}} second ago_plural": "",
-  "Started {{date}}": "",
+  "Started on {{firstWindowStartedAt}}": "",
```

Comment:
Shouldn't the new `Didn\'t start yet` string be here as well?

Reply:
Yes, it was missing! Thanks!